src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -571,6 +571,19 @@ NamespaceOrTypeOrAliasSymbolWithAnnotations createErrorType()
                 return TypeWithAnnotations.Create(CreateErrorType());
             }
         }
+
+        internal static CSDiagnosticInfo? GetNullableUnconstrainedTypeParameterDiagnosticIfNecessary(LanguageVersion languageVersion, in TypeWithAnnotations type)
+        {
+            if (type.Type.IsTypeParameterDisallowingAnnotation())
+            {
+                var requiredVersion = MessageID.IDS_FeatureDefaultTypeParameterConstraint.RequiredVersion();
```

Comment:
>IDS_FeatureDefaultTypeParameterConstraint [](start = 48, length = 41)

This is somewhat confusing because `default` constraint isn't involved.

Reply:
Added a comment.

---
In reply to: [456677882](https://github.com/dotnet/roslyn/pull/45993#discussion_r456677882) [](ancestors = 456677882)